### PR TITLE
fix(mrf): design review results page

### PR DESCRIPTION
**Pull request by KenLSM**
## Problem

Results Page
- align the values in the box
- Date & time displayed on thank you page and results should use last submission timing

**Breaking Changes** 
- No - this PR is backwards compatible  

## Before & After Screenshots
| Item | Before | After |
|--------|--------|--------|
| Results Metadata | Screenshot 2024-04-02 at 1 37 16 AM | Screenshot 2024-04-02 at 12 41 37 AM |

### CODE REVIEW

File: frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx [hunk 1]
```diff
@@ -52,24 +52,29 @@ const LoadingDecryption = memo(() => {
...
 const StackRow = ({
   label,
-  value,
+  children,
   isLoading,
   isError,
 }: {
   label: string
-  value: string
+  children: string | ReactNode
```
Comment (justynoh) [inline frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx]: nit: can remove `string` here

File: frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx [hunk 2]
```diff
@@ -52,24 +52,29 @@ const LoadingDecryption = memo(() => {
...
-        {label}:
-      </Text>
-      <Skeleton isLoaded={!isLoading && !isError}>{value}</Skeleton>
+      <Box minW="8rem">
+        <Text as="span" textStyle="subhead-1" whiteSpace="nowrap">
+          {label}:
+        </Text>
+      </Box>
+      <Skeleton isLoaded={!isLoading && !isError}>
+        <Box>{children}</Box>
```
Comment (justynoh) [inline frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx]: nit: I think no need the box here? 🤔

**Review by justynoh — APPROVED** [review]

**Review by justynoh — CHANGES_REQUESTED** [review]
to check with stacey on reordering behavior first!

**Comment by KenLSM** [comment]
Note: there's a possibility of the submissions being reordered as `lastModified` changes to the latest MRF response time.

**Comment by KenLSM** [comment]
> Note: there's a possibility of the submissions being **reordered** as lastModified changes to the latest MRF response time.

@staceytan1998 need to get your opinions on the re-ordering behaviour for MRF. For storage mode forms, new submissions doesn't affect existing ordering, but for MRF if we now order by `lastModified` (as opposed to `createdAt`), then the ordering for both CSV and Response Results Page will change. 

-- was reviewing our PRs and noticed this.

**Comment by staceytan1998** [comment]
@KenLSM @justynoh is there any way to not reorder for now or is more effort needed to do that? I don't have a good sensing from users as of yet!

**Comment by KenLSM** [comment]
The reorder is caused by us switching the timestamp that is displayed to the admin from `createdAt` to `lastModified`. If we don't do this, then there will be no reordering.

`createdAt`: timestamp of first submission
`lastModified`: timestamp of latest submission

Lmk if this explanation is not clear.

**Comment by KenLSM** [comment]
@staceytan1998 The reorder is caused by us switching the timestamp that is displayed to the admin from `createdAt` to `lastModified`. If we don't do this, then there will be no reordering.

`createdAt`: timestamp of first submission
`lastModified`: timestamp of latest submission

Lmk if this explanation is not clear.

**Comment by KenLSM** [comment]
Closing as we're not getting a good signal from admins on this request.

**Pull request closed without merging**